rero/rero-ils
# Maj circulation ui #724

Automatic action from item barcode is not working.

To test:
- rebase, bootstrap and setup as usual
- go to the admin interface
- tests should work at instance level
- tests does not work as angular project level
- to test the dev server of angular got to ui/admin directory and run `npm i` and `ng serve --configuration proxy`
- note that this interface is not fully working, basically it is not possible to scan an item_barcode
